src/common/constants.cpp

```diff
@@ -29,6 +34,9 @@ uint64_t NextPowerOfTwo(uint64_t v) {
 	v |= v >> 16;
 	v |= v >> 32;
 	v++;
+	if (v_in == 0) {
```

Comment:
Should this not be `v`?

Reply:
Alternatively - we can also do something like `if (v_in > 9223372036854775808ULL)` (since we know that is the highest power of two that can fit in a `uint64_t`, `2^63`)

Reply:
it should!